apps/darts-modernisation/darts-migration/prod.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: darts-api-migration
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-api-migration
+  values:
+    java:
+      ingressHost: darts-api-migration.platform.hmcts.net
+      environment:
+        ENABLE_FLYWAY: true
+        RUN_DB_MIGRATION_ON_STARTUP: true
+        DARTS_API_DB_HOST: "darts-migration-prod.postgres.database.azure.com"
+        DARTS_API_DB_NAME: "darts-migration-prod"
+        DARTS_GATEWAY_URL: https://darts-migration-gateway.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts-migration.apps.hmcts.net
+        DARTS_LOG_LEVEL: DEBUG
+        RESTART_ME: '007'
+      keyVaults:
+        "darts":
+          secrets:
+            - name: migration-POSGRES-USER
+              alias: DARTS_API_DB_USERNAME
+            - name: migration-POSTGRES-PASS
+              alias: DARTS_API_DB_PASSWORD
+            - name: migration-POSTGRES-HOST
+              alias: DARTS_API_DB_HOST
+            - name: migration-POSTGRES-DATABASE
+              alias: DARTS_API_DB_NAME
+    function:
```

Comment:
is duplication needed from the base?

Reply:
its an overwrite - pulling different secrets from the vault for this 'migration'